operations/mimir-mixin/docs/playbooks.md

```diff
@@ -195,6 +195,10 @@ How to **investigate**:
 - Check the `Mimir / Reads` dashboard
   - Looking at the dashboard you should see in which Mimir service the high latency originates
   - The panels in the dashboard are vertically sorted by the network path (eg. gateway -> query-frontend -> query->scheduler -> querier -> store-gateway)
+- Check the `Mimir / Reads resources` dashboard
+  - Check if query frontend pods are at the memory limit - if they are, check in Kubernetes if pods are OOM-ing
+  - Check if querier pods are at the memory limit - if they are, check in Kubernetes if pods are OOM-ing
```

Comment:
I wouldn't mention Kubernetes unless strictly required. We want these runbooks to be useful for non K8S people too. I would just say check if they're OOMing.